ledger/db/src/lib.rs

```diff
@@ -48,10 +48,11 @@ pub use error::Error;
 pub use ledger_trait::{Ledger, MockLedger};
 pub use mc_util_lmdb::{MetadataStore, MetadataStoreError};
 pub use mint_config_store::{ActiveMintConfig, MintConfigStore};
+pub use mint_tx_store::MintTxStore;
 pub use tx_out_store::TxOutStore;
 
 pub const MAX_LMDB_FILE_SIZE: usize = 2usize.pow(40); // 1 TB
-pub const MAX_LMDB_DATABASES: u32 = 24; // maximum number of databases in the lmdb file
+pub const MAX_LMDB_DATABASES: u32 = 29; // maximum number of databases in the lmdb file
```

Comment:
Why 29? AFAIK this PR adds 3 `Database`s.

Reply:
Oops, yeah.